test/e2e_node/cgroup_manager_test.go

```diff
@@ -0,0 +1,70 @@
+/*
```

Comment:
Offline discussed with @dubstack, in the future we'll also check whether the container cgroup is set correctly. Anyway we have to start a container, then we just check the cgroup directory in the container.